rancher/service.go

```diff
@@ -145,7 +146,22 @@ func (r *RancherService) up(create bool) error {
 		service, err = r.context.Client.Service.ActionActivate(service)
 		err = r.Wait(service)
 	}
-
+	for k, v := range r.serviceConfig.Labels {
```

Comment:
Can you change this to `if _, ok := r.serviceConfig.Labels["io.rancher.wait_for_healthcheck"]; ok {`?